how-to-capture-fiddler-trace.md

```diff
@@ -0,0 +1,37 @@
+---
+title: How to capture Fiddler trace
+description: Step by step tutorial for capturing HTTP/HTTPS traffic with Fiddler
+type: how-to
+page_title: How to capture Fiddler trace
```

Comment:
With page titles it's generally a good idea to insert something that indicates where the page is located or what it's a part of. We should discuss if we'd like to have some format for the page title such as "Title | Telerik Platform" or even "Title | Knowledge Base | Telerik Platform"

Should we decide to do that, I believe there's a way to handle it with the layouts so that you don't need to insert it manually.

However, the capitalization comment from above applies.